include/oneapi/dpl/pstl/hetero/dpcpp/execution_sycl_defs.h

```diff
@@ -136,17 +138,19 @@ __get_default_policy_object(DeviceSelector selector)
 }
 static device_policy<> dpcpp_default{__get_default_policy_object(sycl::default_selector{})};
 
-#    if _ONEDPL_FPGA_DEVICE
+#        if _ONEDPL_FPGA_DEVICE
 inline fpga_policy<>&
 __get_fpga_policy_object()
 {
     static fpga_policy<> __sycl_obj{};
```

Comment:
The variable name makes no sense to me - why is it a "SYCL object"? I'd recommend e.g. `__single_base_obj`